src/func-cosmosdb-worker/src/tests/Functional/xxENSONOxx.xxSTACKSxx.Worker.FunctionalTests/Drivers/CosmosDbDriver.cs

```diff
@@ -0,0 +1,123 @@
+    public async Task<bool> IsExistingCosmosDbContainerAsync(string databaseName, string containerName)
+    {
+        try
+        {
+            Database cosmosDbDatabase = _cosmosClient.GetDatabase(databaseName);
+            await cosmosDbDatabase.GetContainer(containerName).ReadContainerAsync();
+            return true;
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine("Error : " + ex.Message);
+            return false;
+        }
+    }
+
+
+    /// <summary>
+    /// Create an item in a Cosmos DB container.
+    /// </summary>
+    /// <param name="databaseName">The name of the CosmosDB database for the container.</param>
+    /// <param name="containerName">The name of the container to create the item in.</param>
+    /// <param name="dbChangeFeedEvent">The item to create in the container.</param>
+    /// <returns>Boolean value to indicate if the operation was successful.</returns>
+    public async Task<bool> CreateItemAsync(string databaseName, string containerName, CosmosDbChangeFeedEvent dbChangeFeedEvent)
+    {
+        try
+        {
+            Database database = _cosmosClient.GetDatabase(databaseName);
+            Container container = await database.GetContainer(containerName).ReadContainerAsync();
+
+            await container.CreateItemAsync(
+                item: dbChangeFeedEvent,
+                partitionKey: new PartitionKey(dbChangeFeedEvent.id)
+            );
+
+            return true;
+
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine("Error : " + e.Message);
+            return false;
+        }
+    }
+
+
+    /// <summary>
+    /// Replace an existing item in a CosmosDB container.
+    /// </summary>
+    /// <param name="databaseName">The name of the CosmosDB database for the container.</param>
+    /// <param name="containerName">The name of the container to replace the item in.</param>
+    /// <param name="updatedDbChangeFeedEvent">The item to replace in the container, matched on ID.</param>
+    /// <returns>Boolean value to indicate if the operation was successful.</returns>
+    public async Task<bool> ReplaceItemAsync(string databaseName, string containerName, CosmosDbChangeFeedEvent updatedDbChangeFeedEvent)
+    {
+        try
+        {
+            var database = _cosmosClient.GetDatabase(databaseName);
+            var container = database.GetContainer(containerName);
+
+            await container.ReplaceItemAsync(
+                item: updatedDbChangeFeedEvent,
+                id: updatedDbChangeFeedEvent.id,
+                partitionKey: new PartitionKey(updatedDbChangeFeedEvent.id));
+
+            return true;
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine("Error : " + e.Message);
+            return false;
+        }
+    }
+
+
+    /// <summary>
+    /// Replace an existing item in a CosmosDB container
```

Comment:
Is this "Delete" ?

Reply:
Yes it is, thanks very much for spotting the typo.
Will correct.